### PR TITLE
in progress entry

### PR DESCRIPTION
**Pull request by clfchris**

### CODE REVIEW

File: in-progress/chfa5693-zed.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+*   Started: 2016-03-01
+*   Completed: TBD
+
+I searched through github repos to find a small open source project to
+contribute to. I know that big projects have a harder time merging small
+users.
+
+## The Change
+
+I changed a misleading and self-sonflicting sentence in the READ file.
```
Comment (colagrosso) [inline in-progress/chfa5693-zed.md]: Replace "READ" with "README" or "README.md".

File: in-progress/chfa5693-zed.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Progress
+
+*   2016-03-01: Today I forked the repo and I am waiting for a response
+*   from the owner. I also looked at the pther pull requests and it seems
```
Comment (colagrosso) [inline in-progress/chfa5693-zed.md]: Replace "pther" with "other".

**Comment by colagrosso** [comment]
You might get some resistance on that PR. I agree with your change, but the author might be sensitive about the way they originally worded it. It will be interesting to see what happens.

**Comment by colagrosso** [comment]
I will merge this to the `in-progress` directory once you correct the typos above.

**Comment by clfchris** [comment]
i have made those minor corrections

**Comment by clfchris** [comment]
I am just generally interested to see some sort of response from the author. This is someone else's work so it feels a little weird to add to it, but I understand the concept of open source.

**Comment by colagrosso** [comment]
You PR now has two files where it should have one. Please remove the `in-progress/chfa5693-zed.md` file. You will probably have to run something like:

```
git rm in-progress/chfa5693-zed.md
```

**Comment by colagrosso** [comment]
> You might get some resistance on that PR

I thought this would be the case, and it was. At least they were nice about it. I suggest thanking the reviewer and closing the pull request.

**Pull request merged**